Extensions/BBText/bbtextruntimeobject-pixi-renderer.js

```diff
@@ -112,3 +112,7 @@ gdjs.BBTextRuntimeObjectPixiRenderer.prototype.getWidth = function () {
 gdjs.BBTextRuntimeObjectPixiRenderer.prototype.getHeight = function () {
   return this._pixiObject.height;
 };
+
+gdjs.BBTextRuntimeObjectPixiRenderer.prototype.setWidth = function(newWidth) {};
+
+gdjs.BBTextRuntimeObjectPixiRenderer.prototype.setHeight = function(newHeight) {};
```

Comment:
Note to me : 
I can clean these lines, it's done [here now](https://github.com/4ian/GDevelop/commit/0d5fbdabc998a3673672dcbac8a6d4ade33a6cd0).